### ISSUE TITLE
On "How to join an existing project" - Point 7 "Keep exploring" - Grammar and some minor improvements.

### ISSUE THREAD

**Issue by RituKapur-Developer** [gvwilson/buildtogether.tech#96]
1) First sentence: Thousands of books have been written about how to write code, but only a few about how to read it ([Spinellis2003] being my favorite). 

>>
could be changed to:
"Thousands of books have been written explaining how to write code, but only a few describe how to read it ([Spinellis2003] being my favorite). "
The second version just feels a bit more interesting to read.

2) In my experience, the best approach to learning your way around a new code base is to trace what happens during typical operation.

>> Please consider replacing "typical" with "a typical".

3) What objects are created and (perhaps most importantly) what permanent changes are made to files or databases, if any? The "step over" button your debugger is a very useful tool for this (Chapter 15), since it allows you ignore low-level details while you're trying to paint a bigger picture.

>> 
could be changed to:
"What objects are created, and (perhaps most importantly) what permanent changes are made to files or databases if any? The "step over" button your debugger is a handy tool for this (Chapter 15) since it allows you to ignore low-level details while you're trying to paint a bigger picture."
Basically, I removed some punctuation errors, "very useful" to "handy", and "you ignore" to "you to ignore".
